picard/options.py

```diff
@@ -0,0 +1,340 @@
+import logging
+
+from PyQt6 import QtCore
+
+from picard.config import (
+    BoolOption,
+    FloatOption,
+    IntOption,
+    ListOption,
+    Option,
+    TextOption,
+)
+from picard.const import MUSICBRAINZ_SERVERS
+from picard.const.defaults import (
+    DEFAULT_AUTOBACKUP_DIRECTORY,
+    DEFAULT_CA_PROVIDERS,
+    DEFAULT_CAA_IMAGE_SIZE,
+    DEFAULT_CAA_IMAGE_TYPE_EXCLUDE,
+    DEFAULT_CAA_IMAGE_TYPE_INCLUDE,
+    DEFAULT_CACHE_SIZE_IN_BYTES,
+    DEFAULT_COVER_IMAGE_FILENAME,
+    DEFAULT_CURRENT_BROWSER_PATH,
+    DEFAULT_DRIVES,
+    DEFAULT_FPCALC_THREADS,
+    DEFAULT_LOCAL_COVER_ART_REGEX,
+    DEFAULT_LONG_PATHS,
+    DEFAULT_MUSIC_DIR,
+    DEFAULT_PROGRAM_UPDATE_LEVEL,
+    DEFAULT_QUERY_LIMIT,
+    DEFAULT_RELEASE_TYPE_SCORES,
+    DEFAULT_REPLACEMENT,
+    DEFAULT_SHOW_MENU_ICONS,
+    DEFAULT_STARTING_DIR,
+    DEFAULT_THEME_NAME,
+    DEFAULT_TOOLBAR_LAYOUT,
+    DEFAULT_TOP_TAGS,
+    DEFAULT_WIN_COMPAT_REPLACEMENTS,
+)
+from picard.i18n import N_
+
+from picard.ui.colors import InterfaceColors
+
+
+class PicardOptions:
```

Comment:
Why the class? Wouldn't it be enought to declare all options on top level here?

Reply:
Because if we import without doing anything with it linters will complain, but that's not definitive, WIP

Reply:
Addressed in https://github.com/metabrainz/picard/pull/2433/commits/9bf11ba0e0988acacfb03868fc5c09840bfe497f